chat-plugins/economy.js

```diff
@@ -552,6 +552,11 @@ exports.commands = {
 		if (targetUser) {
 			avatar = (isNaN(targetUser.avatar) ? "http://" + serverIp + ":" + Config.port + "/avatars/" + targetUser.avatar : "http://play.pokemonshowdown.com/sprites/trainers/" + targetUser.avatar + ".png");
 		}
+		let dev = () => {
+			if (typeof this.user === 'string' && developers.indexOf(toId(this.user)) > -1) return ' (<font color=#980000><b>Hiddencity Dev</b></font>)';
```

Comment:
I may not be a coder, but I know that Hiddencity DEV doesn't make any sense. This looks like some leeched out code to be honest. Plus I may want to ask, what is the use of VIP and Developer for Wisp at this stage?